consensus/hotstuff/cruisectl/config.go

```diff
@@ -1,27 +1,23 @@
 package cruisectl
 
 import (
-	"math"
 	"time"
 )
 
 // DefaultConfig returns the default config for the BlockRateController.
 func DefaultConfig() *Config {
 	return &Config{
-		TargetTransition: EpochTransitionTime{
-			day:    time.Wednesday,
-			hour:   19,
-			minute: 0,
-		},
+		TargetTransition: DefaultEpochTransitionTime(),
 		// TODO confirm default values
 		DefaultProposalDelay: 500 * time.Millisecond,
 		MaxProposalDelay:     1000 * time.Millisecond,
 		MinProposalDelay:     250 * time.Millisecond,
 		Enabled:              true,
-		N:                    600, // 10 minutes @ 1 view/second
-		KP:                   math.NaN(),
-		KI:                   math.NaN(),
-		KD:                   math.NaN(),
+		N_ewma:               5,
```

Comment:
Do we need to tweak this data in future or it's unlikely?

Reply:
I'm planning to make these configurable via flags. I do think we may want to adjust these over time.

Reply:
Then how about adding some examples to show what will change if we alter those values, because it's complicated.